web/blueprints/infrastructure/tables.py

```diff
@@ -1,77 +1,55 @@
 from flask import url_for
 from flask_login import current_user
-from wtforms.widgets import html_params
 
-from web.blueprints.helpers.table import BootstrapTable, Column
+from web.blueprints.helpers.table import BootstrapTable, Column, LinkColumn, \
+    BtnColumn, button_toolbar
 
 
 class SubnetTable(BootstrapTable):
-    def __init__(self, *a, **kw):
-        super().__init__(*a, columns=[
-            Column('id', '#'),
-            Column('description', 'Beschreibung'),
-            Column('address', 'IP'),
-            Column('gateway', 'Gateway'),
-            Column('reserved', 'Reservierte Adressen',
-                   formatter='table.listFormatter'),
-            Column('free_ips_formatted', 'Freie IPs', col_args={
-                'data-sort-name': 'free_ips',
-            }),
-        ], **kw)
+    id = Column("#")
+    description = Column("Beschreibung")
+    address = Column("IP")
+    gateway = Column("Gateway")
+    reserved = Column("Reservierte Adressen", formatter='table.listFormatter')
+    free_ips_formatted = Column("Freie IPs", col_args={
+        'data-sort-name': 'free_ips',
+    })
 
 
 class SwitchTable(BootstrapTable):
-    def __init__(self, *a, **kw):
-        super().__init__(*a, columns=[
-            Column('id', '#'),
-            Column('name', 'Name', formatter='table.linkFormatter'),
-            Column('ip', 'Management IP'),
-            Column('edit_link', 'Editieren', formatter='table.btnFormatter', width=1),
-            Column('delete_link', 'Löschen', formatter='table.btnFormatter', width=1)
-        ], **kw)
-
-    def generate_toolbar(self):
+    id = Column("#")
+    name = LinkColumn("Name")
+    ip = Column("Management IP")
+    edit_link = Column('Editieren', formatter='table.btnFormatter', width=1)
+    delete_link = Column('Löschen', formatter='table.btnFormatter', width=1)
```

Comment:
There should be used a `BtnColumn` here too.

Reply:
Resolved in 6b41f59866d27292d84120ba8391cb042618f21f